bedrock/mozorg/templates/mozorg/MPL/headers/index.html

```diff
@@ -0,0 +1,68 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/base-resp.html" %}
+
+{% block page_title %}{{ _('Mozilla License Headers') }}{% endblock %}
+
+{% block page_desc %}{{ _('Mozilla License Headers') }}{% endblock %}
+
+{% block page_css %}
+    {% stylesheet 'mpl' %}
+{% endblock %}
+
+{% block content %}
+
+<div id="main-content">
+    <h1>Mozilla License Headers</h1>
+
+    <section>
+        <p>This page gives copy-and-pasteable license headers for Mozilla code. When adding license headers to new files to be checked in to the Mozilla source tree, always use the appropriate one of the following.</p>
+        <h2>MPL 2</h2>
+        <p>This license must be used for all new code, unless the containing project, module or externally-imported codebase uses a different license. If you can't put a header in the file due to its structure, please put it in a LICENSE file in the same directory.</p>
+        <pre>
+        /* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The code indenting here is causing the contents of the `<pre>` tags to be indented on the rendered page, which leads some of them to break out of the width of the column:

<img width="739" alt="screenshot 2015-08-26 09 56 27" src="https://cloud.githubusercontent.com/assets/58062/9493840/df5b7486-4bd8-11e5-9002-46ec4ebe5386.png">

It will be ugly in the source, but perhaps these `<pre>` blocks should be 'un-indented'? Or is there another way to avoid this?